scripts/common.sh

```diff
@@ -26,18 +26,18 @@ set -e
 # changes directory. The paths returned by `caller` are relative to it.
 _initial_work_dir=$(pwd)
 
-if [ "$CIRCLECI" ]
+if [[ ${CIRCLECI:-} != "" ]]
```

Comment:
The problem is that this script might be included in another script where the check for unbound variables via `set -u` was enabled (as with the `scripts/ASTImportTest.sh` script). That means `$CIRCLECI` does only have a real value, if the corresponding environment variables was defined. The script fails otherwise with something like `scripts/common.sh: line 29: CIRCLECI: unbound variable`. So doing this strange substitution ensures that `$CIRCLECI` is always defined, even if executed outside circleci. It will just be empty in that case.

Reply:
Ah, right. So it is fine to keep in that way since the variable must to be set:)